content/en/docs/use-cases/bootstrapping/index.md

```diff
@@ -137,15 +137,31 @@ Serve an opt-in or "upsell" modal/interstitial or page to the user offering them
 
 > Consider showing (or linking to) longer descriptions explaining that all users that are able to unlock the current device will be able to access the account at the relying party to ensure that the user is giving fully informed consent.
 
-If the user consents, call `navigator.credentials.create()`, making sure to ask for a [platform authenticator](/), [user verification preferred](#a-note-about-user-verification), [discoverable credentials](/docs/reference/terms/#discoverable-credential) (formerly known as "resident keys"), and passing an [exclude list](https://www.w3.org/TR/webauthn-2/#dom-publickeycredentialcreationoptions-excludecredentials) of existing passkeys for the account:
+If the user consents, call `navigator.credentials.create()` with the options as shown in the example below:
 
 ```js
 navigator.credentials.create({
   publicKey: {
-    rp: { ...},
-    user: { ...},
-    challenge: ...,
+    rp: {
+      // User friendly name of your service
+      name: "Passkeys Developer",
+      // RP identifier (hostname)
+      id: "passkeys.dev"
+    },
+
+    user: {
+      // persistent, unique identifier for the user account in your backend
+      id: Uint8Array.from("0525bc79-5a63-4e47-b7d1-597e25f5caba", c => c.charCodeAt(0)),
+      // user friendly identifier often displayed to the user (e.g. email address)
+      name: "julia@passkeys.dev",
+      // human readable display name, sometimes displayed by the client
+      displayName: "Julia Coleman"
+    },
+    // the challenge is a buffer of cryptographically random bytes generated on your backend
+    // and should be tightly bound to the current user session
+    challenge: Uint8Array.from(`${randomStringFromServer}`, c => c.charCodeAt(0)),
```

Comment:
For the sake of example I might just put a random string here instead of a var statement here.